### DISCUSSION TITLE
Support x-codeSamples annotation in OpenAPI generator

### DISCUSSION THREAD

**Discussion by tothandras** [fuma-nama/fumadocs#531] [Ideas]
It's a common practice to provide generated SDKs to consume an API.
`x-codeSamples` became a standard annotation.

Reference:
- https://mintlify.com/docs/api-playground/openapi/advanced-features#x-codesamples
- https://www.speakeasyapi.dev/guides/openapi/x-codesamples
- https://app.stainlessapi.com/docs/reference/config#open-api-config

**Comment by fuma-nama** [comment]
Good idea, will implement that later

**Reply by fuma-nama** [reply]
Done 👍
